fem/gridfunc.cpp

```diff
@@ -2452,10 +2452,10 @@ void GridFunction::SaveSTL(std::ostream &out, int TimesToRefine)
       }
    }
 
-   cout << "[xmin,xmax] = [" << bbox[0][0] << ',' << bbox[0][1] << "]\n"
-        << "[ymin,ymax] = [" << bbox[1][0] << ',' << bbox[1][1] << "]\n"
-        << "[zmin,zmax] = [" << bbox[2][0] << ',' << bbox[2][1] << ']'
-        << endl;
+   out << "[xmin,xmax] = [" << bbox[0][0] << ',' << bbox[0][1] << "]\n"
+       << "[ymin,ymax] = [" << bbox[1][0] << ',' << bbox[1][1] << "]\n"
+       << "[zmin,zmax] = [" << bbox[2][0] << ',' << bbox[2][1] << ']'
+       << endl;
```

Comment:
Since 'out' is the name of a parameter, this should be `mfem::out`.

Reply:
Yeah, I saw that one.  Do we really want the output to potentially go to 2 different places?

Reply:
Yes, this is just informational message and it is not part of the STL file that we are writing to `out`. I'll fix this, with some other changes I'm making.

Reply:
Done in Veselin's update.